lmdeploy/messages.py

```diff
@@ -180,6 +181,7 @@ class PytorchEngineConfig:
     adapters: Dict[str, str] = None
     max_prefill_token_num: int = 4096
     thread_safe: bool = False
+    prefix_caching: bool = False
```

Comment:
We need to expose this config to API later, shall we use `enable_prefix_caching` for more clear to users?

Reply:
Sure.

Reply:
`PytorchEngineConfig` and cli?